nanofi/src/api/nanofi.cpp

```diff
@@ -205,7 +205,7 @@ flow_file_record* create_ff_object(const char *file, const size_t len, const uin
 }
 
 flow_file_record* create_ff_object_na(const char *file, const size_t len, const uint64_t size) {
-  flow_file_record *new_ff = new flow_file_record;
+  flow_file_record *new_ff = (flow_file_record*) malloc(sizeof(flow_file_record));
```

Comment:
Happy newless year! :)

Reply:
Oops, sorry, must change the delete to free due to avoid heap corruption.